tests/languages/shell-session/command_string_feature.test

```diff
@@ -13,6 +13,8 @@ foo
 bar
 STRING_END
 
+$ echo \'a # '

```

Comment:
Why doesn't this addition result in additional tokens?

Reply:
Good catch! 
(Could have sworn I added the tokens...)